packages/core/src/domain/configuration/experimentalFeatures.ts

```diff
@@ -0,0 +1,26 @@
+let enabledExperimentalFeatures: Set<string>
+
+export function updateExperimentalFeatures(enabledFeatures: string[]): void {
```

Comment:
since you have added extra code to handle some cases, what about adding some tests to document them and  avoid regression?

Reply:
just added some tests.